core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala

```diff
@@ -192,6 +192,16 @@ abstract class CassandraSpec(
             println(s"""Row:${row.getLong("partition_nr")}, ${row.getString("persistence_id")}, ${row.getLong(
               "sequence_nr")}""")
           })
+
+        println("snapshots")
```

Comment:
cleanup

Reply:
?

Reply:
Ah the println? All the logging here is done with println